### ISSUE TITLE
Some epubs are not displayed properly

### ISSUE THREAD

**Issue by Pocokk** [sumatrapdfreader/sumatrapdf#4035]
**SumatraPDF version**
- Version v3.6.15941 64-bit

**Describe the bug**
When I open the file, every page looks weird, like enlarged text and so forth.

**To Reproduce**
1) Download the sample file https://www.warhammerdigital.com/Downloads/Product/ePub%20and%20MOBI%20Extracts/T/60042799001-WHTOW-Eng-Rulebook-EPUB-2024-PREVIEW.epub
2) Open it
3) See how strange looking it is.
 
**Expected behavior**
It should look something like this on this 
[https://www.warhammerdigital.com/Images/Product/AlternativeBL/xlarge/BLPROCESSED-60632799001-WHTOWRulebookEPub02-updated.jpg](preview image)

**File that reproduces the problem**
60042799001-WHTOW-Eng-Rulebook-EPUB-2024-PREVIEW.epub from https://www.warhammerdigital.com/new-exclusive/featured/warhammer-the-old-world-rulebook-epub-2024.html

**Screenshots**
[If applicable, add screenshots to help explain your problem.](https://imgur.com/a/nlmPbUJ)

**Additional context**
I think this isn't a new issue per-se, but I just noticed it now.

**Comment by GitHubRulesOK** [comment]
Yes its an odd way to build an ePub as Edge also has to use a fixed size
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/f7d5b1f5-e3cc-45ba-87e9-71b23436cd99)

**Comment by Pocokk** [comment]
Another sample file, this is a comic book basically, built in epub format:

https://www.blacklibrary.com/Downloads/Product/ePub%20and%20MOBI%20Extracts/M/Extract-Macragges-Honour-2019.epub

**Comment by kjk** [comment]
Yeah, it has CSS that mupdf doesn't quite understands.

It sets gigantic font sizes and gigantic size for the div, placing each string at an absolute position, presumably hoping it'll get scaled down to the size of the page.

**Comment by kjk** [comment]
Reported to mupdf: https://bugs.ghostscript.com/show_bug.cgi?id=707719

**Comment by Pocokk** [comment]
That company Games Workshop tends to release their rulebooks like this, similar format and whatnot, would be nice if the mu folks can try to have a look at it :)

**Comment by GitHubRulesOK** [comment]
MuPDF have decided as "WontFix" and due to the layout I tend to agree it would be hard to use for reflowable but then again eBook can be Fixed format if scale is correct.

**Comment by Pocokk** [comment]
Sad to read/hear, for these type of content the community then have to rely on other methods to consume such media.

Thank you for the feedback!

**Comment by Pocokk** [comment]
Although now that I read the mupdf report page, I got thinking: can I as an individual manually edit said epubs (position:absolute and transform:scale(0.05)) in order to become usable  to some level?

**Comment by GitHubRulesOK** [comment]
@Pocokk there is an eBooks format that publishers use based on XHTMLX e.g. it is an eBook on steroids as single pages and thus used by say DK (Dorian Kingsly) etc. this is probably closest to the ideal HTML format for readers to generate Glossy PDFs like Start Wars etc..

Last time I looked (about a year ago) I ended up with a very complex cmd file but it was tunable for different book layouts so if the source is good it works across 600 pages no problem however unsure if it is still viable. so you would need to do a fair number of tests since Edge no longer does eBook rendering.
The method is very simple **fix the page scale** then print HTML to PDF with Edge (that has newer problems but still works in practice with a mod or two try https://github.com/GitHubRulesOK/MyNotes/blob/master/SMOPs.MD#html) 

the complex command from a year ago is at https://superuser.com/a/1863755/1769247
